src/js/websocket.js

```diff
@@ -16,7 +16,13 @@
  var net = require('net');
  var util = require('util');
  var EventEmitter = require('events').EventEmitter;
- var tls = require('tls');
+ var tls = (function() {
```

Comment:
There is no need for the function. We can do the following:
```
var tls;
try {
  tls = require(...);
} catch (e) {
  tls = {};
}
```

Could you update the PR to remove the function?